src/roles/vsd-upgrade-destroy-new/tasks/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Invoke vsd-destroy role on new VM
```

Comment:
Where is this role being called from? If it is not called from anywhere, remove it.